web_poet/fields.py

```diff
@@ -2,14 +2,17 @@
 ``web_poet.fields`` is a module with helpers for putting extraction logic
 into separate Page Object methods / properties.
 """
+from __future__ import annotations
```

Comment:
Added to get around the cyclic imports so we could have this annotation: `page: web_poet.ItemPage`